data-prepper-expression/src/main/antlr/DataPrepperExpression.g4

```diff
@@ -240,6 +237,8 @@ NOT_IN_SET : SPACE 'not in' SPACE;
 AND : SPACE 'and' SPACE;
 OR : SPACE 'or' SPACE;
 NOT : 'not' SPACE;
+ADD : '+';
+SUBTRACT : '-';
```

Comment:
Two questions:

1. Is it necessary to add the `ADD` token (and operator) as unary operator? i.e. if we get rid of `ADD`, will it break?
2. Shall we give them more indicative names of unary operator? e.g. positive/negative?